### PR TITLE
style : add prettier

### PR DESCRIPTION
**Pull request by andranugraha**
## What's this PR do ? ⭐️
add prettier

## What are the relevant ticket ? ⭐️
https://gteam.atlassian.net/browse/MAIB-43

## Any deployment note (migrations, new envs, rake tasks, cron) ?
added prettier dependency so might need to run npm install

## Definition of Done:

-   [ ] Does this PR introduces a breaking change ?

-   [x] Does this PR passes Lint locally ?

-   [ ] Does this PR requires Database changes ?

-   [ ] Does this PR requires a regression test by QA ?

-   [x] Does this PR adds new dependencies ? (network setup, new 3rd party services)

### CODE REVIEW

**Comment by hilmiwicak** [comment]
Add snippet below on file `./.prettierignore` to pass the build
```
# staging
/.github
/.heroku
```

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+############################
+
+.env
+license.txt
+exports
+*.cache
+build
+.strapi-updater.json
+
+# staging
```
Comment (andranugraha) [inline .prettierignore:116]: @hilmiwicak already exists here

Comment (hilmiwicak) [reply .prettierignore:116]: mantap sorry didn't notice it.

**Review by hilmiwicak — APPROVED** [review]
Good👍

**Review by madewantara — APPROVED** [review]
goodjob!

**Pull request merged**